src/main/java/us/dot/its/jpo/ode/aws/depositor/AwsDepositor.java

```diff
@@ -176,15 +196,20 @@ public void run(String[] args) throws Exception {
 				boolean gotMessages = false;
 
 				while (true) {
-					ConsumerRecords<String, String> records = stringConsumer.poll(CONSUMER_POLL_TIMEOUT_MS);
+					ConsumerRecords<String, String> records = stringConsumer.poll(Duration.ofMillis(CONSUMER_POLL_TIMEOUT_MS));
 					if (records != null && !records.isEmpty()) {
 						for (ConsumerRecord<String, String> record : records) {
 							try {
 								gotMessages = true;
 								if (depositToS3) {
 									depositToS3(s3, record);
-								} else {
+								} else if (destination.equals("firehose")){
 									depositToFirehose(firehose, record);
+								} else if (destination.equals("gcs")) {
+									depositToGCS(gcsStorage, bucketName, record);
+								} else {
+									logger.error("Invalid destination: " + destination);
+									System.exit(1);
 								}
 							} catch (Exception e) {
 								int retryTimeout = 5000;
```

Comment:
```
logger.error("Error depositing to AWS. Retrying in " + retryTimeout / 1000 + " seconds",
										e);
```
Maybe include depositing to GCS besides AWS in the error message?

Reply:
This log statement now references the specified destination rather than 'AWS' being hardcoded.